api/src/main/java/org/pmiops/workbench/cohortbuilder/SubjectCounter.java

```diff
@@ -28,11 +28,28 @@
                     "from `${projectId}.${dataSetId}.person` person\n" +
                     "where\n";
 
-    private static final String UNION_TEMPLATE =
-            "union distinct\n";
+    private static final String CHART_INFO_SQL_TEMPLATE =
+            "select concept1.concept_code as gender, \n" +
+                    "case when concept2.concept_name is null then 'Unknown' else concept2.concept_name end as race, \n" +
+                    "case when DATE_DIFF(CURRENT_DATE, DATE(person.year_of_birth, person.month_of_birth, person.day_of_birth), YEAR) >= 0 and DATE_DIFF(CURRENT_DATE, DATE(person.year_of_birth, person.month_of_birth, person.day_of_birth), YEAR) <= 1 then '< 2' \n" +
```

Comment:
I believe we're not going to have participants younger than 18 for a while, FWIW.